server/broadcast.go

```diff
@@ -57,16 +68,24 @@ func (pi *packetInfo) broadcastAll(msgs []*message.Signed) {
 	defer pi.tracker.mutex.RUnlock()
 
 	sender := pi.tracker.connections[pi.conn]
+	if sender == nil {
+		sender = pi.tracker.findPlayerInPoolSnapshots(pi.conn)
```

Comment:
I think for now we should just log this and return. We have 2 `sender == nil` blocks now.

We can investigate the `findPlayerInPoolSnapshots` logic in a nother PR.

Reply:
bdff93b